ansible/roles_ocp_gitops_workloads/README.md

```diff
@@ -0,0 +1,9 @@
+This is the official home of everyone's OCP Workloads which can be installed using method.
```

Comment:
can we use asciidoc?

because the official documentation is automatically generated using asciidoc and published using github pages. (github worfklow)


So we could include anything in here maybe at some point, if needed.

Reply:
Sure Let me do it.

Reply:
DOne